tftrt/examples/image-classification/image_classification.py

```diff
@@ -574,20 +628,32 @@ def get_frozen_graph(
     if args.num_calib_inputs < args.batch_size:
         raise ValueError('--num_calib_inputs must not be smaller than --batch_size'
             '({} <= {})'.format(args.num_calib_inputs, args.batch_size))
-
+    if args.mode == 'validation' and args.use_synthetic:
+        raise ValueError('Cannot use both validation mode and synthetic dataset')
+    if args.mode != 'validation' and args.mode != 'benchmark':
+        raise ValueError('invalid mode argument (must be validation or benchmark)')
+    
     def get_files(data_dir, filename_pattern):
         if data_dir == None:
             return []
         files = tf.gfile.Glob(os.path.join(data_dir, filename_pattern))
         if files == []:
-            raise ValueError('Can not find any files in {} with pattern "{}"'.format(
-                data_dir, filename_pattern))
+            raise ValueError('Can not find any files in {} with pattern "{}"'.format(data_dir, filename_pattern))
         return files
 
-    validation_files = get_files(args.data_dir, 'validation*')
+    if args.mode == "validation":
+        validation_files = get_files(args.data_dir, 'validation*')
+    else:    
+        val_paths = get_files(args.data_dir, "n*")
```

Comment:
Is it necessary to have the filenames start with `n*`?
Why can't we instead read every file in the directory?